modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCPasswordGrantTest.java

```diff
@@ -115,18 +116,43 @@ public void testGetAccessTokenForPasswordGrant() throws Exception {
         accessToken = response.then().extract().path("access_token");
     }
 
-    @Test(groups = "wso2.is", description = "Retrieve user claims from user-info endpoint", dependsOnMethods =
-            "testGetAccessTokenForPasswordGrant")
-    public void testUserInfoEndpoint() throws Exception {
+    /**
+     * Provide request data to test userInfoEndpoint
+     * @return object with testUserInfoEndpoint method parameters
+     */
+    @DataProvider(name = "userInfoEndpointRequestDataProvider")
+    public Object[][] userInfoEndpointRequestDataProvider() {
 
-        Map<String, String> params = new HashMap<>();
-        params.put("scope", "openid");
+        Map<String, String> getHeaders = new HashMap<>();
+        getHeaders.put("Authorization", "Bearer " + accessToken);
 
-        Map<String, String> headers = new HashMap<>();
-        headers.put("Authorization", "Bearer " + accessToken);
-        headers.put("Content-Type", "application/x-www-form-urlencoded;charset=UTF-8");
+        Map<String, String> postHeaders = new HashMap<>();
+        postHeaders.put("Content-Type", "application/x-www-form-urlencoded;charset=UTF-8");
```

Comment:
Shall we add the content type header without the charset too?

Reply:
Resolve with the commit 

https://github.com/wso2/product-is/pull/11153/commits/02aa294b47a371e6e7b0945ce7ab69abe27f9ddc